assets/css/style.css

```
@@ -1,54 +1,147 @@
     --white-bg-color: #fffff6;
-
 }
 
 /* Applies to whole page */
 * {
     font-family: 'Poppins', sans-serif;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
 }
 
-/* ============================================== */
-
-/* Navigation Styling */
-
-nav {
-    background-color: var(--light-grey-bg-color);
+body {
+    background-color: var(--white-bg-color);
     color: var(--black-font-color);
-    height: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: 100vh;
+}
+
+header {
     width: 100%;
-    padding: 20px;
+    background-color: var(--light-grey-bg-color);
+    padding: 20px 0;
     text-align: center;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
 }
 
 h1 {
     font-size: 48px;
+    margin: 0;
 }
 
-/* ============================================== */
-
 /* Search box styling */
-
 #search-window {
+    width: 100%;
+    max-width: 800px;
     background-color: var(--white-bg-color);
     text-align: center;
+    padding: 20px;
+    margin: 20px 0;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+#search-feature {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 10px;
+}
+
+#search-box {
+    width: 60%;
+    padding: 10px;
+    font-size: 16px;
+    border: 1px solid var(--light-grey-bg-color);
+    border-radius: 4px;
+    margin-left: 10px;
 }
 
 #button-div {
     width: 100%;
 }
 
 button {
-    padding: 5px 20px;
+    padding: 10px 20px;
     margin: 10px;
+    font-size: 16px;
+    color: var(--white-bg-color);
+    background-color: var(--black-font-color);
+    border: none;
+    border-radius: 4px;
     cursor: pointer;
+    transition: background-color 0.3s ease;
 }
 
-#search-feature {
-    padding-top: 20px;
+button:hover {
+    background-color: #333;
+}
+
+/* Dashboard results styling */
+#dashboard-results {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    max-width: 1200px;
+    padding: 20px;
+    margin-bottom: 20px;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    background-color: var(--white-bg-color);
+    border-radius: 4px;
+}
+
+#location-details-container,
+#location-weather-container {
+    width: 45%;
+}
+
+#location-details-container {
+    text-align: center;
 }
 
-/* ============================================== */
+#location-image {
+    width: 100%;
+    height: auto;
+    border-radius: 4px;
+    margin-top: 10px;
+}
+
+#todays-forecast {
+    margin-bottom: 20px;
+}
+
+.forecast-day {
+    border: 1px solid var(--light-grey-bg-color);
+    padding: 10px;
+    margin: 10px 0;
+    border-radius: 4px;
+    background-color: #f9f9f9;
+}
+
+.forecast-day h3 {
+    margin-bottom: 10px;
+}
+
+.forecast-day p {
+    margin: 5px 0;
+}
+
+/* Footer styling */
+footer {
+    width: 100%;
+    background-color: var(--light-grey-bg-color);
+    text-align: center;
+    padding: 10px 0;
+    position: absolute;
+    bottom: 0;
```

```suggestion
    left: 0;
    bottom: 0;
```
To get rid of small gap present in certain viewports.